test/unit/configuration/configuration_parser_test.js

```diff
@@ -0,0 +1,139 @@
+const assert = require('assert');
+const configurationParser = require('./../../../lib/configuration/configuration_parser');
+const fsPromises = require('fs/promises');
+const os = require('os');
+const path = require('path');
+const { v4: uuidv4 } = require('uuid');
+const tempDir = path.join(os.tmpdir(), uuidv4());
+
+describe('Configuration parsing tests', function () {
+
+  before(async function () {
+    await fsPromises.mkdir(tempDir, { recursive: false });
+  });
+
+  after(async function () {
+    await fsPromises.rm(tempDir, { recursive: true, force: true });
+  });
+
+  it('should parse json', async function () {
+    // given
```

Comment:
we don't write this comments normally in the tests

Reply:
I think it will be good to introduce it, but we can discuss it and apply it globally when we decide we would like to use it

Reply:
for now I will leave it as is for the further discussion